simd/simd-memtest/compiler.h

```diff
@@ -13,10 +13,12 @@
 #define RESTRICT restrict
 #endif
 
-#if defined(_OPENMP) && (( __STDC_VERSION__ >= 199901L ) || (__cplusplus >= 201103L ))
+#if (__STDC_VERSION__ >= 199901L) || (__cplusplus >= 201103L)
+#ifdef _OPENMP
```

Comment:
This prevents a spurious warning about not supporting _Pragma if you compile w/o openmp (even when you support `_Pragma`.